embeddings/transformation/Transformation.py

```diff
@@ -0,0 +1,15 @@
+import abc
+from typing import TypeVar, Generic
+from abc import ABC
+
+Input = TypeVar("Input")
+Output = TypeVar("Output")
+
+
+class Transformation(ABC, Generic[Input, Output]):
```

Comment:
do we want to add explicitly something like `StaticEmbeddings(Transformation)`?

Reply:
or rather extract separate step for Embeddings

Reply:
separated model and task